NSS-Day-Cohort-53/giffygram-spazzy-kermits
# Create New Posts

#### Changes Made
1. Modified file `PostCreate.js` to include several `addEventListener`s that hides the post creation form when the closed button is clicked and submit a new post when the submit post button is clicked.
2. Modified file `dataAccess.js` function `fetchPosts()` to sort posts in descending order.
3. Modified file `PostList.js` to include several `addEventListener`s that display the post creation form and re-render the new post in the DOM.​
4. Modified file `PostCreate.css` to add styling to the Post Create Form

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cg-newPosts
    ```
2. Open a new Terminal tab (ctrl+shift+T) and navigate to the server directory.
3. Test app functionality.
    > Create a post    
    > Watch Post Happen
    > Observe that the sidebars don't close when making a post or closing the post creation form
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
